src/StickyParallaxHeader.js

```diff
@@ -29,6 +29,35 @@ import { getSafelyScrollNode, setRef } from './utils';
 const { divide, Value, createAnimatedComponent, event, timing, ValueXY } = Animated;
 const AnimatedScrollView = createAnimatedComponent(ScrollView);
 
+const HeaderBackgroundImage = (props) => {
```

Comment:
The component looks good, but I would like you to move it (including styles) to a separate file under the src/components/HeaderBackgroundImage directory.